R/split_var_call.R

```diff
@@ -0,0 +1,60 @@
+R_SPLIT = function(domain_dataset,max_length_out = 200){
+  out_n = outt = NULL
+
+  #filtering columns > 200
+  char_200 = domain_dataset %>% select_if(~ max(nchar(.)) >= max_length_out)
+
+  #string split function
+  split_var <- function(string,max_length_out = 200) {
+
+    # Pattern spot
+    pattern = names(which.max(table(str_extract_all(string, "[:punct:]|[:blank:]")))) %>%

```

Comment:
The pattern can be used for split, but it will not work as sep in paste function.